src/main/java/org/folio/circulation/domain/policy/OverdueFinePolicy.java

```diff
@@ -2,29 +2,50 @@
 
 import io.vertx.core.json.JsonObject;
 
+import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
 import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
 
 public class OverdueFinePolicy extends Policy {
+  private Boolean ignoreGracePeriodForRecalls;
+  private Boolean countClosed;
 
   private OverdueFinePolicy(String id) {
-    this(id, null);
+    this(id, null, null, null);
   }
```

Comment:
Why is this not part of the unknown sub-class?

Reply:
Moved.